src/main/kotlin/no/nav/familie/ef/iverksett/iverksetting/domene/IverksettData.kt

```diff
@@ -181,7 +182,8 @@ data class VedtaksdetaljerOvergangsstønad(
     override val tilkjentYtelse: TilkjentYtelse?,
     override val tilbakekreving: Tilbakekrevingsdetaljer? = null,
     override val brevmottakere: Brevmottakere? = null,
-    override val vedtaksperioder: List<VedtaksperiodeOvergangsstønad> = listOf()
+    override val vedtaksperioder: List<VedtaksperiodeOvergangsstønad> = listOf(),
+    val avslagÅrsak: AvslagÅrsak? = null
```

Comment:
Burde vi lagre denne på alle type stønader og sende det til DVH for alle stønadstyper? 👀

Reply:
https://github.com/navikt/familie-eksterne-kontrakter/pull/124